### PR TITLE
binance - fetchLastPrices fix

### PR DESCRIPTION
**Pull request by ttodua**
there was some incompleteness/bug in existing `fetchLastPrices`, and now it's exactly how other methods are implemented (i.e. fetchBidsAsks, fetchTickers...)

### CODE REVIEW

File: ts/src/binance.ts [hunk 1]
```diff
@@ -3004,10 +3004,14 @@ export default class binance extends Exchange {
          * @returns {object} a dictionary of [ticker structures]{@link https://docs.ccxt.com/#/?id=ticker-structure}
          */
         await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols);
```
Comment (carlosmiei) [inline ts/src/binance.ts:3007]: hum, something not working here: 
```
 n binance fetchLastPrices '["BTC/USDT:USDT"]'          
Debugger attached.
2023-06-11T16:52:16.769Z
Node.js: v18.14.0
CCXT v3.1.33
binance.fetchLastPrices (BTC/USDT:USDT)
2023-06-11T16:52:26.101Z iteration 0 passed in 6144 ms

{}
```

**Comment by carlosmiei** [comment]
LGTM ✅ 

```
p binance fetchLastPrices '["BTC/USDT"]'                
Python v3.10.9
CCXT v3.1.33
binance.fetchLastPrices(['BTC/USDT'])
{'BTC/USDT': {'baseVolume': None,
              'datetime': None,
              'info': {'price': '25891.99000000', 'symbol': 'BTCUSDT'},
              'price': 25891.99,
              'quoteVolume': None,
              'side': None,
              'symbol': 'BTC/USDT',
              'timestamp': None}}
```
```
p binance fetchLastPrices '["BTC/USDT:USDT"]'          
Python v3.10.9
CCXT v3.1.33
binance.fetchLastPrices(['BTC/USDT:USDT'])
{'BTC/USDT:USDT': {'baseVolume': None,
                   'datetime': '2023-06-11T17:01:02.557Z',
                   'info': {'price': '25859.80',
                            'symbol': 'BTCUSDT',
                            'time': '1686502862557'},
                   'price': 25859.8,
                   'quoteVolume': None,
                   'side': None,
                   'symbol': 'BTC/USDT:USDT',
                   'timestamp': 1686502862557}}
```

**Review by carlosmiei — APPROVED** [review]

**Comment by ttodua** [comment]
Ouch @carlosmiei why did you spend time on this, I was going to address this, couldn't get to PC on time... sry, tnx

**Comment by carlosmiei** [comment]
> Ouch @carlosmiei why did you spend time on this :( I was going to address this, couldn't get to PC on time... sry, tnx

@ttodua don't worry 🙂

**Pull request merged**